docs/Radio-hardware-specifications-for-EdgeTX.md

```diff
@@ -51,16 +66,19 @@
 * I2C EEPROM
 * Bluetooth module via UART
 * IMU (e.g. LSM6DS33)
+* I2C port extender
+  * if port extenders are used for inputs, the interrupt pin has to be connected to an interrupt capable pin of the MCU
 * SpaceMouse module
 
 ---
 
 ## Black-and-white radios
 
 ### Mandatory hardware features (B/W)
-* MCU: STM32F2 and STM32F4 with minimally 512 kByte flash (for new radios only STM32F4 with minimally 1 MB flash)
-  <br/> STM32F2 running at 120 MHz, STM32F4 running at 168 MHz
+* MCU: STM32F4 with minimally 1 MB flash running at 168 MHz
+  * Support for STM32H562/563 will be available from EdgeTX v3 or v2.11
 * SD/microSD card slot or embedded SD NAND (e.g. XTX XTSD04G, minimally 512 MByte)
+  * Starting with STM32H5 all new radios must use SDIO to connect the storage
```

Comment:
* New radios with STM32H5 microcontrollers, should connect the storage via SDIO.

Reply:
maybe "shall", but not "should"